about.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
+</head>
+<body>
+    <h1>Amber Comfort</h1>    
```

Comment:
I recommend adding a `<header>`, `<main>`, and `<footer>` elements with the appropriate elements nested inside those tags. This will create a more meaningful structure for your pages.